arbitrum-docs/partials/_migrate-nft-collections-partial.md

```diff
@@ -0,0 +1,90 @@
+| **ERC-721**  | Non-fungible tokens that represent ownership of a unique asset.         | - [Third-party bridge](#third-party-nft-bridging-solutions) <br/> - [DIY](#do-it-yourself-diy-bridging)                                                            |
+| **ERC-1155** | A newer standard that allows for more flexibility in how NFTs are used. | - [Third-party bridge](#third-party-nft-bridging-solutions)  <br/> - [DIY](#do-it-yourself-diy-bridging)                                                           |
+
+
+<!-- todo: fact check -->
+
+## Native token bridge
+
+This option is suitable for **end-users** who want to independently migrate their **ERC-20 tokens** to Arbitrum. It's *not* the right option if you're trying to migrate other types of tokens, or if you're a developer who wants to batch-migrate all of your projects' user-owned tokens to Arbitrum. <!-- todo: is this true? -->
+
+The [Arbitrum token bridge](https://bridge.arbitrum.io/) is a trustless, permissionless bridge that allows users to transfer **ERC-20 tokens** (not ERC-721 or ERC-1115 tokens) between Ethereum and <a data-quicklook-from="arbitrum-one">Arbitrum One</a> and <a data-quicklook-from="arbitrum-nova">Nova</a> chains. To use the bridge, visit the [Arbitrum Bridge portal](https://bridge.arbitrum.io/) and follow the instructions.
+
+
+## Third-party NFT bridging solutions
+
+This option is suitable for **end-users** and **developers** who want to migrate **non-ERC-20 tokens** (e.g. ERC-721 or ERC-1155 tokens) to one of Arbitrum's chains. There are many things to consider before using a third-party bridging solution, including:
+
+ - Token metadata
+ - Royalty mechanisms
+ - Perception of which chain's NFTs are canonical
+ - Representational clashes if multiple bridges are used for the same collection
+ - etc
+
+Using a third-party bridging solution may introduce new trust assumptions, forcing you to compromise with properties of permissionlessness and/or trustlessness. Consider these tradeoffs carefully when deciding to use any of the below solutions, and proceed with caution. This list is presented in alphabetical order, and does not represent an endorsement or recommendation; it's simply a list of options that are available, shared with the expectation that you'll **do your own research before using them**:
+
+- [Axelar](https://axelar.network/)
+    - Currently live on Arbitrum One for ERC721 and ERC1155
+- [Layer Zero](https://layerzero.network/)
+    - Currently live on Arbitrum One for ERC721 and ERC1155
+- [Wormhole](https://wormhole.com/)
+    - Currently live on Arbitrum One for ERC721
+- [XP Network](https://xp.network/)
+    - Currently live on Arbitrum Nova for ERC721 and ERC1155
+
+
+### Other third-party solutions
+
+There are some solutions that give you the best of both worlds: **canonical representation of foundational NFTs on their original chain**, *and* **Arbitrum's scaling capabilities to facilitate fast and cheap experiences**:
```

Comment:
@amarrazza - does this representation of the "other" section seem solid? Flagging minor editorial edits made to the intro & bullets here

Reply:
I like the structure, will leave comments on specific changes to copy